src/devices_models/devices/common/nodal_expression.jl

```diff
@@ -0,0 +1,89 @@
+struct NodalExpressionSpec
+    forecast_label::String
+    # TODO: Remove this hack
```

Comment:
Should we submit an issue to track this?

Reply:
This hack is part of the Simulation refactor. I will clarify